src/rosdep2/main.py

```diff
@@ -296,6 +296,10 @@ def _rosdep_main(args):
                            "locally ignore a rosdep key is creating a local rosdep rule "
                            "with an empty list of packages (include it in "
                            "/etc/ros/rosdep/sources.list.d/ before the defaults).")
+    parser.add_option("--install-method",
+                      dest='inst_key', default=None,
+                      help="Affects the 'db' verbs. db command print out packages only "
```

Comment:
It should be `Affects the 'db' verb.`, the `verb` shouldn't be plural.

I would reword the second sentence like this: `filters the output of the 'db' command by packages installed with the provided install method; can be one of <installers>`, then I would fill `<installers>` with the actual list of installers. You should be able to build it. Failing that I would work the last bit `... install method; can be any installer, e.g. apt, pip, etc.`.